.github/workflows/build_main.yml

```diff
@@ -0,0 +1,111 @@
+name: facebook/redex/build_main
+on:
+  pull_request:
+    branches:
+    - main
+    - stable
+  push:
+    branches:
+    - main
+  schedule:
+    - cron: 0 0 * * *
+env:
+  CACHE_VERSION: xxxxx1
+jobs:
+  build-20_04:
+    runs-on: 4-core-ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v4.1.1
+    - uses: "./.github/actions/build_debian"
+      with:
+        job_name: ubuntu_20.04
+  build-22_04:
+    runs-on: 4-core-ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4.1.1
+    - uses: "./.github/actions/build_debian"
+      with:
+        job_name: ubuntu_22.04
+  build-deb_oldoldstable:
+    runs-on: 4-core-ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v4.1.1
+    - uses: "./.github/actions/build_debian"
+      with:
+        use_sdk: false
```

Comment:
General question: what is build-deb_oldoldstable?

Also, is the `dx: command not found` error related to use_sdk being set to false here? What happens if it's true?

Reply:
`build-deb_oldoldstable` is meant to mirror the [circleci job here](https://github.com/facebook/redex/blob/main/.circleci/config.yml#L478). It specifies `use_sdk: false` as does the circleci job.

Honestly, I don't know what happens if we change this to `true`. I'm hoping someone with better knowledge of this project can pick up where I left of and get this over the finish line.